### PR TITLE
docs: Improve Contributing Guide

### PR DESCRIPTION
**Pull request by dplewis**
### New Pull Request Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/parse-php-sdk/blob/master/SECURITY.md).
- [x] I am creating this PR in reference to an [issue](https://github.com/parse-community/parse-php-sdk/issues?q=is%3Aissue).

### Approach

* Document `Helper::print`. PHPUnit doesn't allow `echo`, `var_dump`, `print_r` to output to STDOUT
* Remove unused links
* Remove unused cloud code folder
* Add Code of Conduct

### TODOs before merging

- [x] Add changes to documentation (guides, repository pages, in-code descriptions)

### CODE REVIEW

**Review by mtrezza — APPROVED** [review]
Looks good, is this ready for merge?

File: tests/cloudcode/cloud/main.js
Comment (mtrezza) [inline tests/cloudcode/cloud/main.js:1]: Why has this been deleted?

Comment (dplewis) [reply tests/cloudcode/cloud/main.js:1]: It's unused, outdated and confusing for contributors. Those cloud code functions don't run against the test suite.

Comment (mtrezza) [reply tests/cloudcode/cloud/main.js:1]: Got it

**Comment by parseplatformorg** [comment]
🎉 This change has been released in version [2.2.0](https://github.com/parse-community/parse-php-sdk/releases/tag/2.2.0)

**Pull request merged**